ocs_ci/utility/aws.py

```diff
@@ -1270,7 +1271,10 @@ def update_config_from_s3(bucket_name=constants.OCSCI_DATA_BUCKET, filename=cons
         config.update(config_yaml)
         return config_yaml
     except NoCredentialsError:
-        logger.warn('Failed to fetch auth.yaml from ocs-ci-data')
+        logger.warning('Failed to fetch auth.yaml from ocs-ci-data')
+        return None
```

Comment:
the None is redundant here.
```suggestion
        return
```